recipes/newrelic/infrastructure/ohi/postgres/rhel.yml

```diff
@@ -86,6 +88,29 @@ install:
             echo "The infrastructure agent is required to install this integration, we recommend going through our guided install path for this pre-requisite which can be found here:  https://docs.newrelic.com/docs/new-relic-guided-installation-overview" >> /dev/stderr
             exit 1
           fi
+        - |
+          IS_DATABASE_VALID=$(PGPASSWORD={{.NR_CLI_PASSWORD}} psql -U {{.NR_CLI_USERNAME}} -w {{.NR_CLI_DATABASE}} -h localhost -p {{.NR_CLI_PORT}} -c "select version();" 2>&1 | grep -i PostgreSQL | grep -v grep | wc -l)
+          if [ $IS_DATABASE_VALID -eq 0 ] ; then
+            echo "[Error]: The provided database name is not accessible with the provided username and password and port. See https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/postgresql-monitoring-integration#comp-req for more info." >> /dev/stderr
```

Comment:
@Julien4218 Would it be easier to read these messages as they are output if we split the docs link onto another line?

Reply:
We could. I've heard we were thinking of capturing the last line in stderr in our events, so we could know what was the last error. If we do that, we would miss on the root cause error message.